### PR TITLE
[DDW-509] Show success message for saving logs

### PR DESCRIPTION
**Pull request by daniloprates**
This PR implements a success message for logs download.

## Todo:

- [x] Loading page
- [x] Connecting page
- [x] Support page
- [x] Update Copy Address notification to add a close button and the correct background
- [x] Correct copy for the "Logs successfully downloaded" message - @nikolaglumac 
- [x] Japanese translation - @nikolaglumac

## Screenshots:

image

---

### Basics

- [ ] PR is updated to the most recent version of target branch (and there are no conflicts)
- [ ] PR has good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance tests are passing (`yarn run test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn run dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn run package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn run flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn run lint`)
- [ ] Text changes are proofread and approved (Jane Wild)
- [ ] There are no missing translations (running `yarn run manage:translations` produces no changes)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn run storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly documented and commented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-rendering
- [ ] Any code that only works in Electron is neatly separated from components

### Testing
- [ ] New feature / change is covered by acceptance tests
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature / change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review:
- [ ] Merge PR
- [ ] Delete source branch
- [ ] Move ticket to `done` on the Youtrack board

### CODE REVIEW

**Comment by daniloprates** [comment]
@a-rukin what do you think of a different opacity for the close button when hovering the notification area? Here it has .5 and .8 when hovering. 

hover

**Comment by a-rukin** [comment]
@daniloprates good idea! like it :)

File: source/renderer/app/containers/LoadingPage.js [hunk 1]
```diff
@@ -47,23 +64,54 @@ export default class LoadingPage extends Component<InjectedProps> {
...
+    const { actions } = this.props;
+    const fileName = generateFileNameWithTimestamp();
+    const destination = global.dialog.showSaveDialog({
+      defaultPath: fileName,
+    });
+    if (destination) {
+      actions.profile.downloadLogs.trigger({ fileName, destination, fresh: true });
+    }
+    const { id, duration } = this.notification;
+    actions.notifications.open.trigger({ id, duration, });
```
Comment (DominikGuzei) [inline source/renderer/app/containers/LoadingPage.js]: You forgot to check if the downloaded logs are *actually* successfully saved before showing the notification!

File: source/renderer/app/containers/settings/categories/SupportSettingsPage.js [hunk 2]
```diff
@@ -1,46 +1,89 @@
...
+    const { actions } = this.props;
     const destination = global.dialog.showSaveDialog({
       defaultPath: fileName,
     });
     if (destination) {
-      this.props.actions.profile.downloadLogs.trigger({ fileName, destination, fresh: true });
+      actions.profile.downloadLogs.trigger({ fileName, destination, fresh: true });
     }
+    const { id, duration } = this.notification;
+    actions.notifications.open.trigger({ id, duration, });
```
Comment (DominikGuzei) [inline source/renderer/app/containers/settings/categories/SupportSettingsPage.js]: You forgot to check if the downloaded logs are *actually* successfully saved before showing the notification!

File: source/renderer/app/containers/LoadingPage.js (hunk 1 above)
Comment (daniloprates) [reply source/renderer/app/containers/LoadingPage.js]: You are right! Do you think listening to the action is enough or we should actually check if the file exists?

**Comment by a-rukin** [comment]
> Was wondering if a "View in Finder/Open folder" button wouldn't be useful?
This may cause more bad than good.
In any case, a user will get access to the logs. Wise users will know the path by themselves and they will know the risks of changing something. Others can hurt themselves and then start complaining even more, so I'd prefer to avoid giving them a loaded gun 😂

**Comment by daniloprates** [comment]
@a-rukin fair enough :)

**Comment by nikolaglumac** [comment]
@daniloprates do we have translations for this PR?

**Comment by darko-mijic** [comment]
> do we have translations for this PR?
No, I need to sort this out @nikolaglumac

**Comment by nikolaglumac** [comment]
@daniloprates @darko-mijic I suggest we use the following text for the success message: `Logs successfully downloaded`. @darko-mijic please confirm this so that I can send it over for translation 🙏

**Pull request merged**